components/cover/template.rst

```diff
@@ -60,12 +60,16 @@ Configuration variables:
   of hiding one of them. Defaults to ``false``.
 - **has_position** (*Optional*, boolean): Whether this cover will publish its position as a floating point number.
   By default (``false``), the cover only publishes OPEN/CLOSED position.
-  By setting this to true you can also send any position in between using the publish action.
+  By setting this to true you can also send any position in between using the publish action. 
+  Parameter useless if you set the POSITION_ACTION (is set to TRUE).
```

Comment:
We should validate that in code - it's an easy validator to write. The docs don't need to be cluttered with this.

Reply:
Well actually there is not much to validate, if sb sets `has_position` to false and also set `position_action` then it has position overriden to true, this is done in python side. But I don't know how easy will it be to validate since the default value of `has_position` is false, so we'd need to differentiate from default value and explicit false value just to give an error.